portfolio/src/main/webapp/style.css

```diff
@@ -1,60 +1,139 @@
-*{
-    margin: 0;
-    border: 0;
-    padding: 0;
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+* {
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+h2 {
+    font-size: 3vw;
 }
-header{    
+body {
+    line-height: 1.6;
+    width: 80vw;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+header {
     display: flex;
-    justify-content: space-between;
     align-items: center;
-    padding: 0.5%;
+    padding-top: 60px;
+    justify-content: space-between;
+    overflow: hidden;
+}
+
+header strong {
+    font-size: 1vw;
 }
 
-#menu ul {
+header ul {
     margin: 0;
     padding: 0;
     overflow: hidden;
     list-style-type: none;
 }
 
-#menu ul li {
+header ul li {
     font-weight: 500;
     font-size: 16px;
     display: inline-block;
-    margin: 0px 20px;
+    margin: auto 20px;
+    border-bottom: 2px ;
 }
 
-#menu ul li:hover {
-    font-weight: bold;
+header ul li a:hover {
     border-bottom: 2px solid black;
 }
 
-#menu ul li a{
+header ul li a {
     text-decoration: none;
     text-align: center;
     color: black;
+    display: block;
+    padding: 3px;
 }
 
+section {
+    padding-top: 150px;
+    padding-bottom: 150px;
+}
 
 #home {
-    background: red;
-    height: 100vh;
-    width: 99vw;
-    vertical-align: middle;
-}
-
-#about-me{
-    background: orange;
-    height: 100vh;
-    width: 99vw;
-}
-#projects{
-    background: yellow;
-    height: 100vh;
-    width: 99vw;
-}
-#contact-me{
-    background: green;
-    height: 100vh;
-    width: 99vw;
+    display: flex;
+    font-size: 15px;
+    text-align: center;
+    margin: auto;
+    border-bottom: 1px solid black;
+}
+#home #text {
+    margin-top: auto;
+    margin-bottom: auto;
+    text-align: left;
+    width: 50%;
+    float: left;
+}
+
+#denzilpicture {
+    width: 50%;
+    float: right;
+    border-radius: 50%;
+}
+
+#about-me {
+    display: flex;
+}
+
+#about-me #experiences {
```

Comment:
Since IDs are unique on a page, it's enough to reference them with just their name, so `#experiences { ... }` would be enough in this case.

Reply:
There are some rules later in the file that can change, too.